src/genetic_testing/assay_target/primer_design.py

```diff
@@ -0,0 +1,177 @@
+import re
```

Comment:
Can you add a comment here that says where this code is coming from? E.g. a link to the repo

Reply:
Okay, I've added the comments.